src/views/Settings.vue

```diff
@@ -2,6 +2,9 @@
   <ion-page>
     <ion-header :translucent="true">
       <ion-toolbar>
+        <ion-buttons slot="start">
+          <ion-back-button default-href="/home" />
```

Comment:
Is there any issue if we don't enclose the ion-back-button in ion-buttons?

Reply:
No Sir, there are no issues. In-fact the raw HTML is same for both the cases. Used only ion-back-button now.
Thank you sir for reviewing my code.